compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParser.java

```diff
@@ -2859,10 +2862,11 @@ protected STNode parseStatement() {
             case RETRY_KEYWORD:
             case ROLLBACK_KEYWORD:
             case MATCH_KEYWORD:
+            case ON_KEYWORD:
```

Comment:
is `on_keyword` statement start token?

Reply:
No. Fixed with 3189bc6c6a9ee435ca57fa863f10d8e4f357ab6b

Reply:
Fixed with 3189bc6c6a9ee435ca57fa863f10d8e4f357ab6b